src/components/source_editor/LatticeConfigurationDialog.js

```diff
@@ -61,7 +62,15 @@ class LatticeConfigurationDialog extends React.Component {
         return result;
     }
 
+    /**
+     * Takes in a boolean parameter and uses that to set the value of isDisabled
+     * @param param
+     * @returns {Boolean}
+     */
     isDisabled(param) {
+        if (this.state.isNonPeriodic) {
+            return true;
+        }
         // TODO: implement converter from primitive to conventional cells and re-enable editables
         // const lattice = new Made.Lattice(this.state.lattice);
         return false // !lattice.editables[param];
```

Comment:
I can remove lines 74,75 if you want. They were originally in the code and i'm not sure if the comments are still relevant or if we need to save them for the future.